cmd/kogito/command/deploy/deploy_service.go

```diff
@@ -113,33 +72,18 @@ func (i *deployCommand) RegisterHook() {
 		// Args validation
 		Args: func(cmd *cobra.Command, args []string) error {
 			if len(args) > 2 {
-				return fmt.Errorf("requires 1 arg, received %v", len(args))
+				return fmt.Errorf("requires 2 arg, received %v", len(args))
```

Comment:
Well, second argument is optional, service name is the only mandatory parameter.
So I would say the `requires 1 arg` was correct.

Reply:
done